js/blogtini.js

```diff
@@ -279,12 +290,28 @@ async function fetcher(url)  {
 async function main() {
   let tmp
 
-  // see if this is an (atypical) "off site" page/post, compared to the main site
-  // eslint-disable-next-line no-use-before-define
-  const [my_frontmatter] = markdown_parse(document.getElementsByTagName('body')[0].innerHTML)
-  const base = my_frontmatter?.base
+  /**
+   * Pick all content of the page, let's figure out what to do with it.
+   * Beware, if we use innerHTML and want the front-matter not to be with escaped entities
+   * we got to use textContent instead.
+   */
+  const textContent = document.getElementsByTagName('body')[0].textContent
+  const [raw_fm] = splitFrontMatterAndMarkdown(textContent)
+
```

Comment:
I've noticed that `innerHTML` converts things into htmlentities (e.g. YAML with `&`).  Also, maybe make `splitFrontMatterAndMarkdown` just do that: split the page (not shown in this PR)